src/utils/src/arg_parser.rs

```diff
@@ -277,6 +278,13 @@ impl<'a> Arguments<'a> {
             return Ok(());
         }
 
```

Comment:
Can you please remove this newline and also update the comment from lines 271-273 to apply for both `--help` and `--version` flags, something like: ``If `--help` or `--version` is provided as a parameter, we artificially skip the parsing of other command line arguments by adding just that argument to the parsed list and returning.``